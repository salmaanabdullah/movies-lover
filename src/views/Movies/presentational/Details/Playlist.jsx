import React, { memo } from 'react';
import PropTypes from 'prop-types';
import playBtn from 'src/assets/playBtn.svg';
import styles from './style.module.scss';

const Playlist = ({ trailerText }) => (
  <div>
    <div className={styles.videoCard}>
      <img src={playBtn} alt={'play button'} />
      <p className={'mb-0'}>{trailerText}</p>
    </div>
  </div>
);

Playlist.propTypes = {
  trailerText: PropTypes.string,
};

Playlist.defaultProps = {
  trailerText: '',
};

export default memo(Playlist);
