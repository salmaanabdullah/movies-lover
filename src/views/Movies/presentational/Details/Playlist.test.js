import React from 'react';
import { shallow } from 'enzyme';
import Playlist from './Playlist';

it('expect to render movie component', () => {
  expect(shallow(<Playlist />)).toMatchSnapshot();
});
