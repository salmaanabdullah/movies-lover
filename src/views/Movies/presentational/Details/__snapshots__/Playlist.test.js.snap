// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render movie component 1`] = `
<div>
  <div
    className="videoCard"
  >
    <img
      alt="play button"
      src="playBtn.svg"
    />
    <p
      className="mb-0"
    />
  </div>
</div>
`;
